test/verify/files/cockpitMachinesTestExternalProvider.js

```diff
@@ -173,6 +176,18 @@
     },
 
     /**
+     * Call `Send Non-Maskable Interrupt` on the VM.
+     */
+    SENDNMI_VM: function (payload) {
+      console.log('PROVIDER.SENDNMI_VM called with params: ' + JSON.stringify(payload));
+      const vmName = payload.name;
+      const connectionName = payload.connectionName;
+      return function (dispatch) {
+        console.log('SENDNMI_VM not implemented for this PROVIDER');
```

Comment:
If i'm reading this write this shouldn't happen, if the action is not supported we don't show the button. Maybe change to an console.error.